manifests/value.pp

```diff
@@ -1,24 +1,25 @@
-# = Define: registry::value
+# @summary High level abstraction on top of registry_key and registry_value resources
 #
+# @note
```

Comment:
Due to the text not being indented below the tag, it is not put into a `@note` block.